### ISSUE TITLE
New Site Request

### ISSUE THREAD

**Issue by dumketo** [redowan99/Redowan-CloudStream#57]
### Please Add https://moviebox.ng/

**Please ADD both WEbsites**

**Comment by rockhero1234** [comment]
What special in these than existing providers

**Comment by Tawhid5** [comment]
Hey brother, can you add SAM Online FTP server? It will help me.
server link: http://172.16.50.4/
 [
If your server does not support this IP, please contact me. I can help you access this server.
My email: tawhidalmustasin@gmail.com (http://172.16.50.4/)

**Comment by redowan99** [comment]
> Hey brother, can you add SAM Online FTP server? It will help me. server link: http://172.16.50.4/ [ If your ISP does not support this IP, please contact me. I can help you access this server. My email: [tawhidalmustasin@gmail.com](mailto:tawhidalmustasin@gmail.com)

Check your email.

**Comment by Tawhid5** [comment]
Yeah, thanks! It’s working perfectly now—no need for SAM Online. But I was wondering, is there any way to add a Bangla subtitle option? The built-in subtitles on Cloudstream are really slow. Maybe you could create a Streamplay-type extension that includes Discovery Flix and other sources? If that’s possible, it would be awesome!

**Comment by Tawhid5** [comment]
When I Click any video play or download button it's showing "No link found "
